src/sentry/ingest/transaction_clusterer/rules.py

```diff
@@ -194,16 +206,10 @@ def update_rules(project: Project, new_rules: Sequence[ReplacementRule]) -> None
     rule_store.merge(project)
 
 
-def update_redis_rules(project: Project, new_rules: Sequence[ReplacementRule]) -> None:
-    if not new_rules:
-        return
+def bump_last_used(project: Project, pattern: str) -> None:
+    """If an entry for `pattern` exists, bump its last_used timestamp in redis
 
-    last_seen = _now()
-    new_rule_set = {rule: last_seen for rule in new_rules}
-    rule_store = CompositeRuleStore(
-        [
-            RedisRuleStore(),
-            LocalRuleStore(new_rule_set),
-        ]
-    )
-    rule_store.merge(project)
+    The updated last_used timestamps are transferred from redis to project options
+    in the `cluster_projects` task.
+    """
+    RedisRuleStore().update_rule(project, pattern, _now())
```

Comment:
There is a chance that this update overwrites a _newer_ timestamp written by the transaction clusterer, but it can only differ by ~1 second, which is irrelevant for our 90 day expiry window.